sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/AppendBlobAsyncClient.java

```diff
@@ -78,7 +78,9 @@ public final class AppendBlobAsyncClient extends BlobAsyncClientBase {
 
     /**
      * Indicates the maximum number of blocks allowed in an append blob.
+     * @deprecated due to refactoring of code.
```

Comment:
Don't need to mention "due to refactoring of code". Instead redirect users to the getter method and add a comment like "This constant is deprecated. Use #getMaxBlocks() instead."